target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -247,6 +247,11 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "2:lan:3" "3:lan:2" "4:lan:1" "1:wan"
 		;;
+	tplink,archer-d7-v1|\
+	tplink,archer-d7b-v1)
+		ucidef_add_switch "switch0" \
+			"0@eth1" "3:lan:3" "4:lan:2" "5:lan:1" "6@eth0" "2:wan:4" "1:wan"
```

Comment:
"1:wan:5" instead of "1:wan"?

Reply:
I don't know how to name it... It is an internal wan connected to the Broadcom cpu (I can't test it because I don't have an adsl anymore, but it could work if previously configured using the OEM fw)

Reply:
Well, I'm not sure, but if you keep like this, won't you have _two_ "port 1" in luci then?
If it's only internal and there is not external access anyway, I would not initialize it here, but just remove it.